src/App.vue

```diff
@@ -7,13 +7,106 @@
 </template>
 
 <script>
+import { workflowService } from 'workflow-service'
+
 const defaultLayout = 'empty'
 
+const QUERIES = {
+  root: `
+    {
+      workflows {
+        id
+        name
+        status
+        owner
+        host
+        port
+        taskProxies(sort: { keys: ["cyclePoint"] }) {
+          id
+          name
+          state
+          cyclePoint
+          latestMessage
+          task {
+            meanElapsedTime
+            name
+          }
+          jobs(sort: { keys: ["submit_num"], reverse:true }) {
```

Comment:
Should be able to the summary thinggy (is it `stateSummary`) rather than requesting full info on each job.

Reply:
Just had a look at the schema in `cylc.flow.network.schema`, the closest I found was `stateTotals`.

![image](https://user-images.githubusercontent.com/304786/68340446-5ba5f800-014b-11ea-8527-8c3cd6bd87e1.png)

We can use that to give the numbers - which are missing - to the tooltip. I think there was a recent discussion on riot where @hjoliver and @dwsutherland mentioned something changes to the backend that I think is the same thing you are suggesting here. (I think.)

Reply:
Here's the endpoint we used to call:

https://github.com/cylc/cylc-flow/blob/e3deb7d7049117e7c2a487f558f8b839a7d2ab25/lib/cylc/network/httpserver.py#L328

From memory I think it gave state totals and the five most recent task-jobs in each state?

Reply:
That'd be handy for the UI. Maybe we should open a new issue in Cylc UI Server - I think - to work on this?

Reply:
This has been resolved below, with some follow-up needed. I'll update the main gscan issue #139 description.